CRM/Utils/HttpClient.php

```diff
@@ -98,7 +98,13 @@ public function fetch($remoteFile, $localFile) {
     }
     curl_setopt($ch, CURLOPT_FILE, $fp);
 
-    curl_exec($ch);
+    if ($this->isRedirectSupported()){
+      curl_exec($ch);
+    }
+    else{
+      $this->curl_exec_follow($ch);
```

Comment:
Thank you!

How would you feel about removing the call to `curl_exec()` and simply using `curl_exec_follow()` all the time? It seems like the`curl_exec_follow()` would work regardless of whether `open_basedir` is configured, and having one path makes it easier to test/support.